doc/user_guide/eds.rst

```diff
@@ -360,6 +360,24 @@ database:
 The lines are returned in order of closeness to the specified energy, and can
 be limited by additional, optional arguments.
 
+Mass absorption coefficient database
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A mass absorption coefficient database (Chantler2005 see http://physics.nist.gov/ffast) is available
```

Comment:
Worth noting I've rewritten the EDS documentation in #965, so yes, the bibliography is the right way to go.